src/services/pcn-packetcapture/src/Packetcapture_dp_ingress.c

```diff
@@ -186,7 +191,14 @@ static __always_inline int handle_rx(struct CTXTYPE *ctx, struct pkt_metadata *m
     }
 
   }
-
+  
+  /* Getting packet timestamp */
+  uint64_t *pkt_timestamp = packet_timestamp.lookup(&key);
```

Comment:
Add a comment such as "The algorithm for timestamping packets is described in file XX function YY"